src/reporting/tezos/TezosConseilClient.ts

```diff
@@ -77,7 +78,8 @@ export namespace TezosConseilClient {
     export async function getAccount(serverInfo: ConseilServerInfo, network: string, accountID: string): Promise<any[]> {
         const query = ConseilQueryBuilder.setLimit(ConseilQueryBuilder.addPredicate(ConseilQueryBuilder.blankQuery(), 'account_id', ConseilOperator.EQ, [accountID], false), 1);
 
-        return getTezosEntityData(serverInfo, network, ACCOUNTS, query);
+        const r = await getTezosEntityData(serverInfo, network, ACCOUNTS, query);
+        return r[0];
```

Comment:
We should validate that at least one result was returned, else return an explicit error.

Reply:
setLimit(1) will guarantee 0 or 1 results